### ISSUE TITLE
More Descriptive "Reference" Tags

### ISSUE THREAD

**Issue by tedmoore** [flucoma/learn-website#275]
After reading @tremblap's comments on [here](https://github.com/flucoma/learn-website/issues/274#issuecomment-1080473779), I was doing a little poking around the learn infrastructure and I had a thought. Sorry it's somewhat tedious but I think it's a good change.

I propose that we make [these](https://learn.flucoma.org/reference/) headers a little more descriptive, simply:

* Analyse Data
* Analyse Sound
* Buffer Utilities
* Decompose Audio
* Slice Audio
* Transform Audio (or, perhaps even better "Hybridise Audio")

The last three in particular because we're not saying what is being decomposed, sliced, or transformed, and then there are acronyms that many people won't be familiar with (NMF, HPSS) so I think we could make it slightly less opaque.

I looked at doing it just now, but it seems it would require changing many tags in the headers of many files, so I just wanted to (1) see if there's an easier / better approach and (2) make sure we agree on the headings before proceeding.

<img width="651" alt="Screenshot 2022-03-28 at 12 40 56" src="https://user-images.githubusercontent.com/1531144/160390501-c89e3956-d81a-4d14-9a3b-34ba10552d35.png">

**Comment by jamesb93** [comment]
It requires manually changing those attributes of the front matter. That should be the only change required and the site will build that page accordingly.

I am not particularly tied to any of these, however, they are aligned with the categories found in the Max documentation so any changes on this side should be reflected elsewhere.

**Comment by tedmoore** [comment]
Cool. that should be easy enough. 

To be clear, I think the categories are good. I'm just saying when we list them, having some nouns present may be clearer for users who aren't as familiar with what kinds of nouns tend to go with these verbs.

What about this I found in `index.svx`? Also easy, but just asking.
 
<img width="628" alt="Screenshot 2022-03-28 at 13 07 53" src="https://user-images.githubusercontent.com/1531144/160394526-efd69edc-98a3-4dfb-a477-19ea03304572.png">

**Comment by jamesb93** [comment]
> Cool. that should be easy enough.
> 
> To be clear, I think the categories are good. I'm just saying when we list them, having some nouns present may be clearer for users who aren't as familiar with what kinds of nouns tend to go with these verbs.
> 
> What about this I found in `index.svx`? Also easy, but just asking.
> 
> <img alt="Screenshot 2022-03-28 at 13 07 53" width="628" src="https://user-images.githubusercontent.com/1531144/160394526-efd69edc-98a3-4dfb-a477-19ea03304572.png">

You're totally right, those keys will also need updating!

```js
references.forEach(r => { categorised[r.category].push(r) });
```

Loads up those arrays with the corresponding references.

**Comment by jamesb93** [comment]
and some nounage could be great - agree also

**Comment by jamesb93** [comment]
I've addressed this in eea52fae96c1b1aab2b4f8d395558da9f168f62f. I think it is much clearer now what the categories mean. To boot, I fixed an issue I wasn't aware of to do with the layout of references updating on the reference page.
